src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Rule Name="Build"
```

Comment:
@mel Ah, here's the problem. When building dotnet/project-system we will generate a BuildPropertyPage.cs file to the obj directory, and it will contain a type with whatever name you provide here.

However, we already have a type named "Build", and that ends up conflicting with the one in the generated file. Rename this to "BuildPropertyPage" and that should take care of at least one of the errors.

Reply:
@melytc you could add a unit test to prevent this problem in future.

Reply:
Well, if you run into this problem you won't be able to build, so the unit test probably doesn't help much. :-)

Reply:
This was it, thanks Tom. Closing this PR.